dont expose social accesstokens when using include=xxx api

```
Himanshus-MacBook-Air:oneauth himanshu$ npm start

> oneauth@0.4.9 start /Users/himanshu/Desktop/oneauth
> node src/server.js

/Users/himanshu/Desktop/oneauth/node_modules/newrelic/lib/config/index.js:1181
    throw new Error(
    ^

Error: Unable to read configuration file /Users/himanshu/Desktop/oneauth/newrelic.js. A default
configuration file can be copied from /Users/himanshu/Desktop/oneauth/node_modules/newrelic/lib/config/default.js
and renamed to 'newrelic.js' in the directory from which you'll be starting
your application.
    at initialize (/Users/himanshu/Desktop/oneauth/node_modules/newrelic/lib/config/index.js:1181:11)
    at Function.getOrCreateInstance (/Users/himanshu/Desktop/oneauth/node_modules/newrelic/lib/config/index.js:1221:23)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/node_modules/newrelic/lib/logger.js:18:34)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! oneauth@0.4.9 start: `node src/server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the oneauth@0.4.9 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging outputabove.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/himanshu/.npm/_logs/2018-05-11T07_05_33_823Z-debug.log
```

Thank you @championswimmer . A new error sadly,
```
oneauth himanshu$ npm start

> oneauth@0.4.9 start /Users/himanshu/Desktop/oneauth
> node src/server.js

module.js:472
    throw err;
    ^

Error: Cannot find module './secrets.json'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/config.js:52:20)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/src/db/models.js:5:16)
```
secrets-sample file is present. Do i need to copy it in secrets.json?

```
Himanshus-MacBook-Air:oneauth himanshu$ npm start
> oneauth@0.4.9 start /Users/himanshu/Desktop/oneauth> node src/server.js

sequelize deprecated String based operators are now deprecated. Please use Symbol based operators for better security, read more at http://docs.sequeli
zejs.com/manual/tutorial/querying.html#operators node_modules/sequelize/lib/sequelize.js:236:13
internal/modules/cjs/loader.js:573
    throw err;
    ^

Error: Cannot find module 'bcrypt'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:571:15)
    at Function.Module._load (internal/modules/cjs/loader.js:497:25)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/src/utils/password.js:4:16)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/src/passport/strategies/user/strategy-local.js:8:19)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/src/passport/strategies/user/index.js:5:23)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/src/passport/passporthandler.js:6:24)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/himanshu/Desktop/oneauth/src/server.js:8:18)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:719:10)
    at startup (internal/bootstrap/node.js:228:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:576:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! oneauth@0.4.9 start: `node src/server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the oneauth@0.4.9 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/himanshu/.npm/_logs/2018-05-15T11_53_39_136Z-debug.log
```
tried:

```
npm install node-gyp -g
npm install bcrypt -g
npm install bcrypt -save
```
Doesn't work!

--- a/src/routers/api/users.js
+++ b/src/routers/api/users.js
@@ -21,19 +21,19 @@ router.get('/me',
                 for (ia of includedAccounts) {
                     switch (ia) {
                         case 'facebook':
-                            includes.push({model: models.UserFacebook, attributes:{exclude:['accessToken']}})
+                            includes.push({ model: models.UserFacebook, attributes: {exclude: ["accessToken","refreshToken"]}})
                             break
                         case 'twitter':
-                            includes.push({model: models.UserTwitter, attributes:{exclude:['token']}})
+                            includes.push({ model: models.UserTwitter, attributes: {exclude: ["token","tokenSecret"]}})
                             break
                         case 'github':
-                            includes.push({model: models.UserGithub, attributes:{exclude:['token']}})
+                            includes.push({ model: models.UserGithub, attributes: {exclude: ["token","tokenSecret"]}})
                             break
                         case 'google':
-                            includes.push({model: models.UserGoogle, attributes:{exclude:['token']}})
+                            includes.push({model: models.UserGoogle, attributes: {exclude: ["token","tokenSecret"]}})
                             break
                         case 'lms':
-                            includes.push(models.UserLms)
+                            includes.push({ model: models.UserLms, attributes: {exclude: ["accessToken"]})
                             break
                     }
                 }
@@ -71,19 +71,19 @@ router.get('/me/address',
                 for (ia of includedAccounts) {
                     switch (ia) {
                         case 'facebook':
-                            includes.push({model: models.UserFacebook, attributes:{exclude:['accessToken']}})
+                            includes.push({ model: models.UserFacebook, attributes: {exclude: ["accessToken","refreshToken"]}})
                             break
                         case 'twitter':
-                            includes.push({model: models.UserTwitter, attributes:{exclude:['token']}})
+                            includes.push({ model: models.UserTwitter, attributes: {exclude: ["token","tokenSecret"]}})
                             break
                         case 'github':
-                            includes.push({model: models.UserGithub, attributes:{exclude:['token']}})
+                            includes.push({ model: models.UserGithub, attributes: {exclude: ["token","tokenSecret"]}})
                             break
                         case 'google':
-                            includes.push({model: models.UserGoogle, attributes:{exclude:['token']}})
+                            includes.push({model: models.UserGoogle, attributes: {exclude: ["token","tokenSecret"]}})
                             break
                         case 'lms':
-                            includes.push(models.UserLms)
+                            includes.push({ model: models.UserLms, attributes: {exclude: ["accessToken"]})
                             break
                     }
                 }

